app/assets/stylesheets/unimported/email.css

```diff
@@ -0,0 +1,87 @@
+body.email-body {
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  color: #3f3f3f;
+  background-color: #f1f1f1;
+  padding: 10px 24px
+}
+
+.dashboard-td {
+    font-size: 125%;
+    width: 33%;
+    padding: 5px;
+    border: 1px solid #ccc;
+}
+
+.dashboard {
+  background: #e1e2e3;
+  padding: 20px 40px;
+  margin: 5px 0px 10px;
+}
+
+.hosts {
+  background: #fff;
+  padding: 10px 20px;
+  border: 1px solid #e1e2e3
+}
+
+.email-td {
```

Comment:
Move next to the headers/table classes?

Why use the `email-` prefix here, but not on other classes?  Would it make sense to either drop the prefix or apply the class only to the `table` element and then define `.email-table < td` or similar?